src/chocolatey/infrastructure.app/templates/NuspecTemplate.cs

```diff
@@ -59,7 +59,7 @@ public class NuspecTemplate
     <authors>__REPLACE_AUTHORS_OF_SOFTWARE_COMMA_SEPARATED__</authors>
     <!-- projectUrl is required for the community feed -->
     <projectUrl>https://_Software_Location_REMOVE_OR_FILL_OUT_</projectUrl>
-    <!--<iconUrl>http://cdn.rawgit.com/[[MaintainerRepo]]/master/icons/[[PackageNameLower]].png</iconUrl>-->
+    <!--<iconUrl>http://rawcdn.githack.com/[[MaintainerRepo]]/master/icons/[[PackageNameLower]].png</iconUrl>-->

```

Comment:
I think we should point to the [Icon Guidelines](https://chocolatey.org/docs/create-packages#package-icon-guidelines) here rather than specifying a specific service. 

/cc @gep13 @ferventcoder

Reply:
I agree. @PeterDaveHello if you can point to this, that would allow us to pull this in.